src/index.js

```diff
@@ -6,5 +6,60 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Customer from "./Customer"
 
-console.log('This is the JavaScript entry file - your code begins here.');
+const roomNumber = document.getElementById('roomNumber')
+const dateStayed = document.getElementById('dateStayed')
+const totalCost = document.getElementById('totalSpent')
+const date = document.querySelector('.date')
+const roomType = document.querySelector('.roomType')
+
+const allCustomersData =
+  fetch("http://localhost:3001/api/v1/customers")
+    .then(response => response.json())
+
+const allRoomsData =
+  fetch("http://localhost:3001/api/v1/rooms")
+    .then(response => response.json())
+
+const allBookingsData =
+  fetch("http://localhost:3001/api/v1/bookings")
+    .then(response => response.json())
+
+Promise.all([allCustomersData, allBookingsData, allRoomsData])
+  .then((allData) => {
+    const customer = new Customer(allData[0].customers[0])
```

Comment:
It's likely you will need to reference customer again in the global context since it houses most of the properties/methods in your app. May want to declare it globally "let customer;" and reassign it on this line.